packages/functions/src/texture-resize.ts

```diff
@@ -2,6 +2,7 @@ import ndarray from 'ndarray';
 import { lanczos2, lanczos3 } from 'ndarray-lanczos';
 import { getPixels, savePixels } from 'ndarray-pixels';
 import type { Document, Transform, vec2 } from '@gltf-transform/core';
+import { listTextureSlots } from '@gltf-transform/functions';
```

Comment:
```suggestion
import { listTextureSlots } from './list-texture-slots';
```

Minor — I think this may need to be a relative import since the target is in the same package.

Reply:
You're right; surprisingly both seem to work.  Let's go with the relative version.